src/participant/participant.ts

```diff
@@ -83,6 +84,7 @@ export default class ParticipantController {
   _docsChatbotAIService: DocsChatbotAIService;
   _telemetryService: TelemetryService;
   _playgroundResultProvider: PlaygroundResultProvider;
+  _copilotStatusSubscription?: vscode.Disposable;
```

Comment:
Is this ever assigned to something? I can only see it being disposed, but never assigned.

Reply:
ah this was left over from a previous approach to the workaround that was here; needs to be removed.